lib/dor/workflow/client/status.rb

```diff
@@ -39,43 +39,12 @@ def initialize(druid:, version:, lifecycle_routes:)
           @druid = druid
           @version = version.to_s
           @lifecycle_routes = lifecycle_routes
-        end
-
-        # @return [Hash{Symbol => Object}] including :status_code and :status_time
-        def info
-          @info ||= begin
-            accessioned_milestones = current_milestones.select { |m| m[:milestone] == 'accessioned' }
-            if accessioned_milestones.any?
-              # if we have an accessioned milestone, this is the last possible step and should be the status regardless of timestamp
-              { status_code: STEPS['accessioned'], status_time: accessioned_milestones.last[:at].utc.xmlschema }
-            else
-              status_code = 0
-              status_time = nil
-              # for each milestone in the current version, see if it comes at the same time or after the current 'last' step.
-              # if so, make it the last and record the date/time
-              current_milestones.each do |m|
-                m_name = m[:milestone]
-                m_time = m[:at].utc.xmlschema
-                next unless STEPS.key?(m_name) && (!status_time || m_time >= status_time)
-
-                status_code = STEPS[m_name]
-                status_time = m_time
-              end
-
-              { status_code: status_code, status_time: status_time }
-            end
-          end
-        end
```

Comment:
This was a public method, but as far as I can tell, it's only ever called within the class, and its purpose is to set two bits of state: a status code and a status time, which will both come from the the latest accessioned milestone or the latest current milestone (or default values if no current milestones). This commit dumps the method and sets the status code and status time values as part of instance initialization.